rgeo_test.go

```diff
@@ -526,6 +526,37 @@ func ExampleRgeo_ReverseGeocode_city() {
 	// Output: <Location> Sapporo, Hokkaidō, Japan (JPN), Asia
 }
 
+func ExampleRgeo_ReverseGeocode_Snapping() {
+	r, err := New(Provinces10)
+	if err != nil {
+		// Handle error
+	}
+
+	loc, err := r.ReverseGeocodeSnapping([]float64{141.5439, 40.5588})
+	if err != nil {
+		// Handle error
+	}
+
+	fmt.Println(loc)
+	// Output: <Location> Aomori, Japan,
+}
+
+func ExampleRgeo_ReverseGeocode_Snapping_NoCountry() {
```

Comment:
I think only one example is needed, but adding a table test with the different possibilities and errors is needed.